guides/v2.3/frontend-dev-guide/layouts/xml-manage.md

```diff
@@ -475,6 +475,86 @@ class Product
 }
 ```
 
+## Manage the 'My Account' dashboard navigation links
+
+You can remove navigation links from the 'My Account' dashboard on the storefront by setting the `remove` attribute.
+
+```xml
+<!-- ################################## -->
+<!-- Magento version: Community Edition -->
```

Comment:
Hi @atwixfirster. That's a good clarification, however, there are Open Source and Commerce Magento versions for the time being. Please, adjust these headings. Thank you.

Reply:
**fixed**

Could you please verify now, @rogyar ?

Thank you!